Cortex/cortex_module/client/__init__.py

```diff
@@ -0,0 +1,28 @@
+import requests
+from requests_ratelimiter import LimiterAdapter
+
+from cortex_module.client.auth import CortexAuthentication
+from cortex_module.client.retry import Retry
```

Comment:
I suggest using the default Retry class from requests
```suggestion
from requests.adapters import Retry
```